packages/babel-preset-gatsby/src/index.js

```diff
@@ -40,6 +43,18 @@ module.exports = function preset(_, options = {}) {
     }
   }
 
+  const plugins = []
+
+  if (PRODUCTION) {
```

Comment:
I think we could target just `` stage === `build-javascript` `` here

Reply:
Thanks, I will test, probably will keep both conditions.

Reply:
`stage === `build-javascript` is the only condition you need as it's only run in production mode ^^

Reply:
@wardpeet & @pieh refactored in [this](https://github.com/gatsbyjs/gatsby/pull/14987/commits/83c5645932bf6403f8fc4a82d792f319672c1a16) commit.
But tomorrow will try to do one more change in the tests - remove the `beforeEach` don't think it makes sense since its only one test. Maybe then the `Gatsby Build Service` gets unstuck 😄

Reply:
Don't worry about `Gatsby Build Service` (that's not required :) )